VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/switch-on-camera-microphone/switch-on-camera-microphone.component.ts

```diff
@@ -72,15 +73,25 @@ export class SwitchOnCameraMicrophoneComponent extends ParticipantStatusBaseDire
     }
 
     async requestMedia() {
-        this.mediaAccepted = await this.userMediaStreamService.requestAccess();
-        this.userPrompted = true;
-        if (!this.mediaAccepted) {
-            this.logger.warn(`[SwitchOnCameraMicrophone] - ${this.participantName} denied access to camera.`, {
-                conference: this.conferenceId,
-                participant: this.participantName
+        this.userMediaStreamService.currentStream$
+            .pipe(take(1))
+            .pipe(timeout(300))
```

Comment:
This is .3 seconds just want to confirm it is intentionally this low

Reply:
Yeah this is intentionally low. It could be increased and might be best to be increased but at this stage no work should need doing as injecting the services starts the work off in the background.

Reply:
I have increased to one second to be on the safe side.